examples/dreambooth/train_dreambooth_lora.py

```diff
@@ -70,6 +70,38 @@
 logger = get_logger(__name__)
 
 
+def text_encoder_lora_state_dict(text_encoder):
```

Comment:
Also, I see that this exact same function is copied over 4 times. So, let's move it to `training_utils.py` please.

Reply:
I think the idea to move it here was because the original function is being deprecated. So I assume this function will be removed eventually. If that's so, do we want to add a `# TODO remove in diffusers vX.Y`? Not sure if there is such a convention in diffusers.

Reply:
Yeah good idea to add:
`# TODO remove ...` statements

@sayakpaul we should try to remove this function once training scripts are changed to use PEFT, I don't want to move them back into src/diffusers